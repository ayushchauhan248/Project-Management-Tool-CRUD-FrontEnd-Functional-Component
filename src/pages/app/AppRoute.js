import "./Approute.css";
import { Route } from "react-router-dom";
import Sidebar from "../../components/shared/Sidebar";
import CreateProject from "./CreateProject";
import Myprofile from "./MyProfile";
import Navbar from "../../components/shared/Navbar";
import ProjectIdPage from "./ProjectIdPage";
import EditPage from "./EditPage";
import React, { Component } from "react";
import DashboardPage from "./DashboardPage";

class AppRoute extends Component {
  render() {
    return (
      <div>
        <Navbar />
        <div id="grid-container">
          <Sidebar></Sidebar>
          <div id="cont">
            <Route path="/dashboard">
              <DashboardPage></DashboardPage>
            </Route>
            <Route path="/create">
              <CreateProject></CreateProject>
            </Route>
            <Route path="/myprofile">
              <Myprofile></Myprofile>
            </Route>
            <Route
              exact
              path="/project/:id"
              render={({ match }) => <ProjectIdPage id={match.params.id} />}
            ></Route>
            <Route path="/edit/:id">
              <EditPage></EditPage>
            </Route>
          </div>
        </div>
      </div>
    );
  }
}

export default AppRoute;
