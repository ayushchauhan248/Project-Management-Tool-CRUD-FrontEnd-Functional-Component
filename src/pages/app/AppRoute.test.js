import { shallow } from "enzyme";
import React from "react";
import AppRoute from "./AppRoute";

it("should render snapshot  of approute", () => {
  const component = shallow(<AppRoute />);
  console.log(component.debug());
  expect(component).toMatchSnapshot();
});
