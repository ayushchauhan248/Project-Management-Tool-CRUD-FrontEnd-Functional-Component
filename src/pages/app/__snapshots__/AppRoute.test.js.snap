// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot  of approute 1`] = `
<div>
  <Navbar />
  <div
    id="grid-container"
  >
    <Sidebar />
    <div
      id="cont"
    >
      <Route
        path="/dashboard"
      >
        <DashboardPage />
      </Route>
      <Route
        path="/create"
      >
        <Connect(CreateProject) />
      </Route>
      <Route
        path="/myprofile"
      >
        <Myprofile />
      </Route>
      <Route
        exact={true}
        path="/project/:id"
        render={[Function]}
      />
      <Route
        path="/edit/:id"
      >
        <Connect(EditPage) />
      </Route>
    </div>
  </div>
</div>
`;
